examples/camera/camera_coloured_ball_detect.py

```diff
@@ -0,0 +1,44 @@
+from os import environ
+from pitopcommon.current_session_info import get_first_display
+from pitop.camera import Camera
+from pitop.processing.algorithms import BallDetector
+from signal import pause
+from pitop.processing.utils.vision_functions import import_opencv
+
+
+cv2 = import_opencv()
+
+environ["DISPLAY"] = get_first_display()
```

Comment:
Would be nice to review the common library soon - it was never intended to be used explicitly by users in this way. This is also only needed if run over SSH, and the solution to that is to run it with this environment variable set. I am not a huge fan of including it like this, although I do see how it is useful.

Reply:
Shall I remove it?

Reply:
i think that we should probably add a section to the docs that explains the extra steps to take when you are running code over SSH. that way we can keep the code minimal, as if it's being run from the device - which would mean removing it here

Reply:
This has been moved to a separate issue: https://github.com/pi-top/pi-top-Python-SDK/issues/325